### DISCUSSION TITLE
Issue with _isApprovedOrOwner() in erc721

### DISCUSSION THREAD

**Discussion by d4r3-w0lf** [Cyfrin/foundry-full-course-cu#2621] [General]
When i use this  _isApprovedOrOwner()  function in my nft contract it is raising an error undeclared identifier. there is no such method in erc721.sol also.

What's the issue or any modifications?

**Comment by d4r3-w0lf** [comment]
Can you help me with this?

**Reply by EngrPips** [reply]
Hey @Nandeesh-D, Please check here https://github.com/Cyfrin/foundry-full-course-cu/blob/main/chronological-updates.md find lesson 11 and see what is written there

**Comment by d4r3-w0lf** [comment]
Thanks, I'll check it out.

**Reply by EngrPips** [reply]
You are welcome.
